src/ProductConstructionService/ProductConstructionService.Api/Metrics/TelemetryRecorder.cs

```diff
@@ -45,10 +30,11 @@ public void SetSuccess()
         public void Dispose()
         {
             _stopwatch.Stop();
-            histogram.Record(_stopwatch.ElapsedMilliseconds);
-            telemetryClient.TrackEvent($"job.{_telemetryName}.{GetSuccessfulString()}");
+            customDimensions.Add("Successful", _successful.ToString());
+            customDimensions.Add("Duration", _stopwatch.ToString());
```

Comment:
Not Ellapsed? What does this give us?

Reply:
same thing as `_stopwatch.Elapsed.ToString()`

Reply:
What is that? `00:00:00` or a number? Is it seconds/ms or what?

Reply:
Yeah, this is bad. Let's have a rounded `long` of milliseconds so that we can perform math operations on it easily

Reply:
updated to milliseconds